### ISSUE TITLE
Create a new endpoint for text similarity check only

### ISSUE THREAD

**Issue by iamyajat** [IEEE-VIT/auto-grader#3]
Right now, one endpoint exists which takes in an image with the correct answers and returns the marks.
Create a new endpoint that will take in the student's answers with the correct answers in a JSON and return the marks.

**Comment by Pretty-19** [comment]
Hi @iamyajat  can you assign this issue to me?

**Comment by iamyajat** [comment]
@Pretty-19 sure

**Comment by Pretty-19** [comment]
@iamyajat   unassigned issue open to others

**Comment by amitpant14** [comment]
Hi @iamyajat can you assign this issue to me?

**Comment by iamyajat** [comment]
@amitpant14 sure

**Comment by amitpant14** [comment]
@iamyajat Yes, I followed the instructions from the readme. There seem to be two issues currently.
1. When I run the uvicorn command to launch the API, it gets stuck after starting the reloader. 
2. It also recommends installing indexer module, which is not compatible with python 3.
Hence I am not able to launch the API locally for any further work.

**Comment by iamyajat** [comment]
Which version of Python are you using?
